Makefile

```diff
@@ -128,7 +128,7 @@ endif
 # Example:
 #   make run
 run: build
-	$(OUT_DIR)/local/go/bin/openshift start
+	hack/run-openshift.sh start
```

Comment:
> $(OUT_DIR)/local/bin/$(shell bash -c 'source hack/common.sh; os::build::host_platform')/openshift start ?

I'll grant you that its not the most beautiful one liner, but I think it gets the job done.